### PR TITLE
prevent data modification during data / cluster migrations

### PR DESCRIPTION
**Pull request by snopoke**
remembered that this never got done after the eNikshay migration

### CODE REVIEW

File: corehq/form_processor/reprocess.py [hunk 1]
```diff
@@ -30,6 +31,10 @@ class ReprocessingError(Exception):
 
 
 def reprocess_unfinished_stub(stub, save=True):
+    if any_migrations_in_progress(stub.domain):
+        logger.info("Ignoring stub during data migration: %s", stub.xform_id)
+        return
+
```
Comment (mkangia) [inline corehq/form_processor/reprocess.py]: What does this prevent?

**Review by mkangia — APPROVED** [review]
Dont have context on what all should be prevented but the changes look fine.

**Review by proteusvacuum — APPROVED** [review]

File: corehq/form_processor/reprocess.py (hunk 1 above)
Comment (snopoke) [reply corehq/form_processor/reprocess.py]: This prevents unifinished submission stubs from being re-processed.

Comment (mkangia) [reply corehq/form_processor/reprocess.py]: Cool, Thanks!

**Pull request merged**